.github/CODEOWNERS

```diff
@@ -1,4 +1,5 @@
 *    @HarryMichal @debarshiray
 /data/gfx/*.gif    @jimmac
+/images/archlinux  @Foxboron
```

Comment:
I see that `ID=arch` in `os-release(5)` on Arch Linux.  To avoid confusion, I think we should also name the directory `arch`.

Reply:
I took the liberty to change the directory name to `arch`.